### ISSUE TITLE
Ignore Structure before \begin{document}?

### ISSUE THREAD

**Issue by nathanatgit** [James-Yu/LaTeX-Workshop#3947]
## Pre-checks*
- [x] The feature request has not been suggested in this repository.

## The Missed*
I'm drafting a thesis containing multiple tex files.
The current structure is
```
\documentclass[12pt,a4paper,twoside]{book}
\input{packages.tex}
\input{styCtrl.tex}
\begin{document}
\input{preface/cover_Ph.D.tex} 
\include{chapters/chpt1.tex}
...
\end{document}
```
The structure view show extra empty chapters and subsections because of style definition in `\input{styCtrl.tex}`
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/20399271/51ca7aad-cc4d-4e7a-8529-2f911c9e14f3)
and currently i can't find a way to ignore them by configuring `latex-workshop.view.outline.sections` or `latex-workshop.view.outline.commands`

## The Solution*
Maybe removing the structure detection before \begin{document} would be a reasonable cure.
Or can i add some ignore control settings in `latex-workshop.view.outline.sections` or `latex-workshop.view.outline.commands` to remove certain tags since `section|alternative` can detect multiple tags, would it be possible that `section!alternative` to remove mismatched tags.

**Comment by James-Yu** [comment]
Fixed in ec159b5

**Comment by sdhosseini88** [comment]
> Fixed in [ec159b5](https://github.com/James-Yu/LaTeX-Workshop/commit/ec159b57f40d41f112361b25697784ce26f28c47)

Is this fix downloadable from VS Code Extensions?
